Cargo.toml

```diff
@@ -19,5 +19,14 @@ panic = 'abort'
 codegen-units = 1
 
 [patch.crates-io]
+zcash_address = { path = "components/zcash_address" }
+zcash_client_backend = { path = "zcash_client_backend" }
+zcash_client_sqlite = { path = "zcash_client_sqlite" }
 zcash_encoding = { path = "components/zcash_encoding" }
+zcash_extensions = { path = "zcash_extensions" }
+zcash_history = { path = "zcash_history" }
 zcash_note_encryption = { path = "components/zcash_note_encryption" }
+zcash_primitives = { path = "zcash_primitives" }
+zcash_proofs = { path = "zcash_proofs" }
```

Comment:
Remember to remove these once zcashd is compiling.

Reply:
These are not necessary at all for `zcashd` compilation. I've removed them.